assets/base.css

```diff
@@ -116,6 +116,7 @@ body {
   min-height: 100%;
   width: 100%;
   overflow-x: hidden;
+  word-break: break-word;
```

Comment:
Can we avoid applying this globally to whole body and only apply it in the div where this issue is happening?

If we apply it globally we could get weird line breaks in the middle of the word in places where they are not necessary.

Reply:
Yes, we can but, I think, we should apply this property globally because we can't predict what words and where the client wants to use. For example, the Hero section looks broken if `break-word` is not applied on mobile
![Arc_2024-06-25 12-23-48@2x](https://github.com/booqable/chameleon-theme/assets/40244261/f33427bd-2a2e-4f06-ab0c-fe0f4cf117b2)
![Arc_2024-06-25 12-23-14@2x](https://github.com/booqable/chameleon-theme/assets/40244261/d389d100-9883-4590-b20b-74ab20e8333d)

Reply:
I was reading up a bit on MDN, it seems that `word-break: break-word` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#syntax). It can also cause issues with some languages.

Maybe we can use [oveflow-wrap: break-word](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap) instead?

Reply:
Make sense